### DISCUSSION TITLE
copilot error - At this time, Copilot is not available in your location

### DISCUSSION THREAD

**Discussion by zarabaat** [community/community#53091] [Copilot Conversations]
### Select Topic Area

Bug

### Body

hi, in this few weeks i get this error about my github copilot when i lunch my IDE,
![image](https://user-images.githubusercontent.com/124144642/232694195-1c534113-4d24-4643-a2d5-6dacde6999bc.png)

i try to logout and login back again in my account and this don't help,
and again try to change my billing info on my account to set location again and this is not working as well

can someone help me with that?

**Comment by Akash1134** [comment]
Hi @zarabaat ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can continue to use other GitHub services including your Pro plan. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear.

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!
